### ISSUE TITLE
4.2.2 broken on Node

### ISSUE THREAD

**Issue by mastermatt** [nodeshift/opossum#392]
## Node.js Version:
v12.13.0

## Operating System:
 18.7.0 Darwin Kernel Version 18.7.0
Same results one Docker node:12, node:12-alpine, node:12-slim

## Steps to Produce Error:
Attempting to import the lib as a dependency errors with:
```
.../node_modules/opossum/dist/webpack:/circuitBreaker/webpack/universalModuleDefinition:10
})(window, function() {
 ^
ReferenceError: window is not defined
...
```

I assume this is related to https://github.com/nodeshift/opossum/pull/384 (webpack - set entrypoint to dist/opossum.js)
fyi: @tavogel @lholmquist @lance

**Comment by mastermatt** [comment]
It seems #384 removed the non-transpiled version of the published code, which is what Node was using. The transpiled code was only meant for browsers (I hope) 😢 

Can that PR just be reverted?

**Comment by mastermatt** [comment]
I've confirmed this is an issue on Node 8 and 10 too.

**Comment by mNalon** [comment]
Same to me :(

**Comment by lholmquist** [comment]
well crap.  I've reverted that commit and published a new version 4.2.3-0 under the "canary" tag in npm,  @mNalon @mastermatt, would you mind trying that version,  if it is ok,  then i'll release a real updated version with that commit #384 reverted.

**Comment by mNalon** [comment]
It is working for me!

**Comment by mastermatt** [comment]
The canary tag works in my libs also.

**Comment by myvipbackup2** [comment]
Same to me

**Comment by lholmquist** [comment]
ok,  let me release it as a proper release

**Comment by lholmquist** [comment]
fixed released as 4.2.3
